app/views/provisioning_templates/_custom_tabs.html.erb

```diff
@@ -11,7 +11,7 @@
   <%= snippet_message(@template) %>
   <div id="association" <%= display? @template.snippet %> >
     <%= how_templates_are_determined %>
-    <%= multiple_selects f, :operatingsystems, Operatingsystem, @operatingsystems.try(:map, &:id), :label => _("Applicable<br>Operating Systems").html_safe %>
+    <%= multiple_selects f, :operatingsystems, Operatingsystem, @operatingsystems.try(:map, &:id), :label => _("Applicable Operating Systems").html_safe %>
```

Comment:
nitpick: if the string doesn't contain html now, there is no need to mark it as html_safe any more